ticketbuyer.go

```diff
@@ -0,0 +1,93 @@
+package main
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/decred/dcrd/chaincfg/chainhash"
+	"github.com/decred/dcrutil"
+	"github.com/decred/dcrwallet/ticketbuyer"
+	"github.com/decred/dcrwallet/waddrmgr"
+	"github.com/decred/dcrwallet/wallet"
+	"github.com/decred/dcrwallet/wtxmgr"
+)
+
+// startTicketPurchase launches ticketbuyer to start purchasing tickets.
+func startTicketPurchase(w *wallet.Wallet, ticketbuyerCfg *ticketbuyer.Config) {
+	retryDuration := time.Second * 5
+	for {
+		if w.ChainClient() != nil {
+			break
+		}
+		time.Sleep(retryDuration)
+		tkbyLog.Debugf("Retrying chain client connection in %v", retryDuration)
+	}
+	dcrdClient := w.ChainClient().Client
+	walletCfg := &ticketbuyer.WalletCfg{
```

Comment:
yeah, we can afford some code duplication during the transition, that's no problem.